### PR TITLE
Use same `stof` logic in both `CutPlaneMenu` and `CameraControl`

### PR DESCRIPTION
**Pull request by rondoor124**
Hi @pablo-mayrgundter, I finished issue https://github.com/bldrs-ai/Share/issues/544.
Please check it and give me feedback.

### CODE REVIEW

File: src/utils/strings.js [hunk 1]
```diff
@@ -58,3 +58,21 @@ export function findUrls(str) {
...
+ * @param {string} str Input string.
+ * @return {number} float.
+ */
+export function stof(str) {
+  const floatDigits = 2
+  const val = parseFloat(parseFloat(str).toFixed(floatDigits))
+  if (isFinite(val)) {
+    const rounded = parseFloat(val.toFixed(0))
+    return rounded === val ? rounded : val
+  } else {
```
Comment (pablo-mayrgundter) [inline src/utils/strings.js:74]: Now that this is a util, it should throw instead of warn.

Please do that and update jsdoc as well.

File: src/utils/strings.js [hunk 2]
```diff
@@ -58,3 +58,21 @@ export function findUrls(str) {
...
 }
+
+/**
+ * Convert string to finite float.
+ *
+ * @param {string} str Input string.
+ * @return {number} float.
+ */
+export function stof(str) {
+  const floatDigits = 2
```
Comment (pablo-mayrgundter) [inline src/utils/strings.js:69]: pls move this up to a param `len=3`; I added a comment asking for the default to be changed from 2 to 3:

https://github.com/bldrs-ai/Share/issues/544

File: src/utils/strings.js [hunk 3]
```diff
@@ -58,3 +58,21 @@ export function findUrls(str) {
     }
   })
 }
+
+/**
+ * Convert string to finite float.
```
Comment (pablo-mayrgundter) [inline src/utils/strings.js:63]: I think this is a little more specific.  Here's an updated API.   Please also add unit tests for these and some boundary conditions with NaN, etc.

```
/**
 * Convert a string-encoded float to a truncated float, of fixed-length `len` or no decimal point expansion
 * if equal to zero.  e.g. with `len=3`:
 *  - '12.34567' -> 12.345
 *  - '12.340' -> 12.34 // same with '12.34'
 *  - '12.300' -> 12.3   // same with '12.3'
 *  - '12.000' -> 12 
 */

File: src/utils/strings.js [hunk 4]
```diff
@@ -58,3 +58,21 @@ export function findUrls(str) {
...
+ *
+ * @param {string} str Input string.
+ * @return {number} float.
+ */
+export function stof(str) {
```
Comment (pablo-mayrgundter) [inline src/utils/strings.js:68]: pls rename `floatStrTrim`

**Review by pablo-mayrgundter — CHANGES_REQUESTED** [review]

File: src/utils/strings.js (hunk 1 above)
Comment (rondoor124) [reply src/utils/strings.js]: i have done.

File: src/utils/strings.js (hunk 3 above)
Comment (rondoor124) [reply src/utils/strings.js]: i have done.

File: src/utils/strings.js (hunk 4 above)
Comment (rondoor124) [reply src/utils/strings.js]: i have done.

**Review by pablo-mayrgundter — APPROVED** [review]
